[2022-08-25 11:19:05,277] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: taxi_rides_data.create_taxigov_raw_table manual__2022-08-25T11:19:03.318024+00:00 [queued]>
[2022-08-25 11:19:05,288] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: taxi_rides_data.create_taxigov_raw_table manual__2022-08-25T11:19:03.318024+00:00 [queued]>
[2022-08-25 11:19:05,288] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-25 11:19:05,288] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-25 11:19:05,288] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-25 11:19:05,311] {taskinstance.py:1397} INFO - Executing <Task(PostgresOperator): create_taxigov_raw_table> on 2022-08-25 11:19:03.318024+00:00
[2022-08-25 11:19:05,317] {standard_task_runner.py:52} INFO - Started process 7326 to run task
[2022-08-25 11:19:05,320] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'taxi_rides_data', 'create_taxigov_raw_table', 'manual__2022-08-25T11:19:03.318024+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/taxi_rides_dag.py', '--cfg-path', '/tmp/tmpm2nos442', '--error-file', '/tmp/tmptxgipojv']
[2022-08-25 11:19:05,322] {standard_task_runner.py:80} INFO - Job 67: Subtask create_taxigov_raw_table
[2022-08-25 11:19:05,389] {task_command.py:371} INFO - Running <TaskInstance: taxi_rides_data.create_taxigov_raw_table manual__2022-08-25T11:19:03.318024+00:00 [running]> on host ddc8002727e0
[2022-08-25 11:19:05,505] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=taxi_rides_data
AIRFLOW_CTX_TASK_ID=create_taxigov_raw_table
AIRFLOW_CTX_EXECUTION_DATE=2022-08-25T11:19:03.318024+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-25T11:19:03.318024+00:00
[2022-08-25 11:19:05,518] {base.py:68} INFO - Using connection ID 'redshift_default' for task execution.
[2022-08-25 11:19:07,098] {dbapi.py:231} INFO - Running statement: 
    CREATE TABLE IF NOT EXISTS raw__taxigov_corridas (
        base_origem VARCHAR(200) NOT NULL,
        qru_corrida INTEGER NOT NULL,	
        nome_orgao VARCHAR(200),	
        status_corrida VARCHAR(200),
        motivo_corrida VARCHAR(200),
        km_total DECIMAL(16,2),
        valor_corrida DECIMAL(16,2), 
        data_abertura TIMESTAMP,
        data_despacho TIMESTAMP,
        data_local_embarque	TIMESTAMP,
        data_inicio	TIMESTAMP,
        data_final TIMESTAMP,
        origem_endereco	 VARCHAR(200), 
        origem_bairro  VARCHAR(200), 
        origem_cidade  VARCHAR(200), 
        origem_uf  VARCHAR(200), 
        destino_solicitado_endereco  VARCHAR(200), 
        destino_efetivo_endereco  VARCHAR(200), 
        origem_latitude	DECIMAL(16,2),
        origem_longitude DECIMAL(16,2),
        destino_solicitado_latitude	DECIMAL(16,2),
        destino_solicitado_longitude DECIMAL(16,2),
        destino_efetivo_latitude DECIMAL(16,2),
        destino_efetivo_longitude DECIMAL(16,2),
        ateste_setorial_data TIMESTAMP,
        conteste_info VARCHAR(900)
    )
    DISTSTYLE AUTO;, parameters: None
[2022-08-25 11:19:07,710] {postgres.py:94} INFO - INFO:  Relation "raw__taxigov_corridas" already exists and will be skipped

[2022-08-25 11:19:07,749] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=taxi_rides_data, task_id=create_taxigov_raw_table, execution_date=20220825T111903, start_date=20220825T111905, end_date=20220825T111907
[2022-08-25 11:19:07,818] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-25 11:19:07,873] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
